demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementFromRowBuilderWithZipSupport.php

```diff
@@ -209,11 +260,33 @@ public function setFeedback(Cell $cell): ?ConstraintViolationListInterface
 
     public function setNumberOfAnonymVotes(Cell $cell): ?ConstraintViolationListInterface
     {
-       return $this->baseStatementFromRowBuilder->setNumberOfAnonymVotes($cell);
+        return $this->baseStatementFromRowBuilder->setNumberOfAnonymVotes($cell);
+    }
+
+    public function setFileReferences(?Cell $cell): ?ConstraintViolationListInterface
```

Comment:
Again, as to my knowledge this method will never be called with a `null` parameter.

Reply:
done in ae76ffe5a4f3439ed179fcd37d6cbb974151a2c2